languages/en-US/dashboard.json

```
@@ -1,19 +1,19 @@
 {
-    "FIRST_LOGIN": "{{user}} connected for the first time to the dashboard! :tada:",
+    "FIRST_LOGIN": "{{user}} logged on to the dashboard for the first time! :tada:",
     "NOT_FOUND": "Oops! Page not found.",
     "NOT_FOUND_CONTENT": "We did not find the page you were looking for. In the meantime, you can go back to the dashboard or try using the search form.",
     "ERR_OCCURRED": "Oops! Something went wrong.",
     "ERR_OCCURRED_CONTENT": "We will try to rectify this immediately. In the meantime, you can go back to the selector or try using the search form.",
-    "SEARCH": "Search servers...",
+    "SEARCH": "Searching for servers...",
```

Ici c'est un input "Chercher des serveurs..." et non "Recherche des serveurs..."
```suggestion
    "SEARCH": "Search for servers...",
```